bin/gwd/gwDaemon.ml

```diff
@@ -338,9 +334,19 @@ let propose_base conf =
   Hutil.trailer conf
 
 let general_welcome conf =
-  match Util.open_etc_file "index" with
-    Some ic -> Util.html conf; Templ.copy_from_templ conf [] ic
-  | None -> propose_base conf
+  include_template conf [] "index"
+    (fun () ->
+      let title _ = Wserver.printf "Base" in
+      Hutil.header conf title;
+      Wserver.printf "<ul><li>";
+      Wserver.printf "<form method=\"get\" action=\"%s\">\n" conf.indep_command;
+      Wserver.printf "<input name=\"b\" size=\"40\"> =&gt;\n";
+      Wserver.printf
+        "<button type=\"submit\" class=\"btn btn-secondary btn-lg\">\n";
+      Wserver.printf "%s" (capitale (transl_nth conf "validate/delete" 0));
+      Wserver.printf "</button>\n";
+      Wserver.printf "</li></ul>";
+      Hutil.trailer conf)
```

Comment:
What is it?

Reply:
Affichage d'une page d'accueil en l'absence du nom d'une base. template index.txt
Cela permet à un site d'héberger plusieurs bases (hgouraud.fr par exemple)

Reply:
Ok mais ça n'a rien à voir avec la PR du coup...

Reply:
yes and no. The `include_template conf [] "index"` is now bracketed by the begin/end function!
If I understand correctly, the code in ` (fun () -> .. ` is what is being displayed if index.txt is not present!